From 58701635c8e12a514d63c1771699ec42514c59f3 Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Thu, 23 Aug 2018 17:18:32 +0200
Subject: [PATCH] make/project.mk: no -Werror

That's a bit strict for user code (including the Arduino core).
---
 make/project.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make/project.mk b/make/project.mk
index 10556b1..8508a31 100644
--- a/make/project.mk
+++ b/make/project.mk
@@ -266,7 +266,7 @@ EXTRA_CPPFLAGS ?=
 CPPFLAGS := -DESP_PLATFORM -D IDF_VER=\"$(IDF_VER)\" -MMD -MP $(CPPFLAGS) $(EXTRA_CPPFLAGS)
 
 # Warnings-related flags relevant both for C and C++
-COMMON_WARNING_FLAGS = -Wall -Werror=all \
+COMMON_WARNING_FLAGS = -Wall \
 	-Wno-error=unused-function \
 	-Wno-error=unused-but-set-variable \
 	-Wno-error=unused-variable \
-- 
2.7.4

